warehouse/templates/accounts/two-factor.html

```diff
@@ -112,6 +120,13 @@ <h2>{% trans %}Authenticate with an app{% endtrans %}</h2>
             <input type="submit" value="{% trans %}Verify{% endtrans %}" class="button button--primary">
           </div>
 
+          <div class="form-group">
+            <label>
+              <input type="checkbox" name="remember_device" value="true" />
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏does this input also need an `id`? I noticed it differed from the webauthn one

Reply:
It don't think it needs an `id` because I'm not referencing it in JS or CSS or anywhere else.

Reply:
The `remember_device_webauthn` input is being referenced in `webauthn.js`.